eng/Versions.props

```diff
@@ -157,6 +157,9 @@
     <SystemSecurityPermissionsPackageVersion>9.0.0-preview.7.24366.18</SystemSecurityPermissionsPackageVersion>
     <SystemTextEncodingCodePagesPackageVersion>9.0.0-preview.7.24366.18</SystemTextEncodingCodePagesPackageVersion>
     <SystemTextJsonPackageVersion>9.0.0-preview.7.24366.18</SystemTextJsonPackageVersion>
+    <!-- This is a minimum version for various projects to work. It's used for netfx-targeted components that run in Visual Studio
+         because in those cases, Visual Studio is providing System.Text.Json, and we should work with whichever version it ships. -->
+    <SystemTextJsonBasePackageVersion>8.0.4</SystemTextJsonBasePackageVersion>
```

Comment:
Bumping the netfx reference to 8.0.4 will fail to satisfy this constraint for VS 17.11. That's bad, right?